packages/esm-ward-app/src/routes.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://json.openmrs.org/routes.schema.json",
+  "backendDependencies": {
+    "webservices.rest": "^2.2.0"
+  },
+  "extensions": [
+    {
+      "component": "root",
```

Comment:
It's definitely unorthodox to have the same component as both an extension and a page.